schema/workflow.json

```diff
@@ -521,24 +531,15 @@
         }
       ]
     },
-    "eventref": {
+    "produceeventref": {
```

Comment:
I would take this opportunity to rename action top level properties to something like:

```yaml
actions:
  - name: my-action
    call: {} #instead of functionRef, maybe in another PR?
    pub: {} #or publish, or produce, instead of produceEventRef
    sub: {} #or subscribe, or consume, instead of consumeEventRef
```

Reply:
I think call should be done with another PR
I think we can use publish and subscribe, combined what which I commented in https://github.com/serverlessworkflow/specification/pull/826#discussion_r1505827718, so rather than name, we have eventRef property (so a publish that refers to event xxxx, a subscribe that refers to event xxxX)

Reply:
I think that we should do it in this PR, as we are altering the structure of an action anyways, let's do it right this time then.

`publish` and ` subscribe` sound perfect to me.

I replied to your comment, in its tread 😉

Reply:
Regarding call, I really think is better to have smaller PRs, just to keep discussion smaller ;)

Reply:
Let's change it here, I think it's reasonable 👍

Reply:
I would say this should be a new issue explaining why we want to change funcionRef to call or functionCall. If we do so, we need to purge the word reference from the spec.

Reply:
Just created https://github.com/serverlessworkflow/specification/issues/828 to address it.